pkg/config/ca/ca.go

```diff
@@ -89,8 +89,8 @@ func Update(jaeger *v1.Jaeger, commonSpec *v1.JaegerCommonSpec) {
 		ReadOnly:  true,
 	}
 
-	commonSpec.Volumes = append(commonSpec.Volumes, volume)
-	commonSpec.VolumeMounts = append(commonSpec.VolumeMounts, volumeMount)
+	commonSpec.Volumes = util.RemoveDuplicatedVolumes(append(commonSpec.Volumes, volume))
```

Comment:
So, if a volume was defined at the common spec, it takes precedence over this one here, right?

Reply:
Yes